### PR TITLE
Fix layout evaluation ver2

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

**Comment by vuducluan** [comment]
OK

File: app/views/admin/evaluations/_form.html.erb [hunk 1]
```diff
@@ -5,26 +5,35 @@
     <div class="col-md-8">
       <%= @user.name %><br>
     </div>
-    <%= render "shared/error_messages", object: @evaluation %>
   </div>
-  <div>
-    <div class="col-md-6">
+    <%= render "shared/error_messages", object: @evaluation %>
```
Comment (yongsokheng) [inline app/views/admin/evaluations/_form.html.erb]: thua indent

**Comment by thiennv57** [comment]
OK

**Comment by yongsokheng** [comment]
OK

**Pull request merged**